tests/testImageDecorrelation.py

```diff
@@ -54,25 +55,26 @@ def singleGaussian2d(x, y, xc, yc, sigma_x=1., sigma_y=1., theta=0.):
     return out
 
 
-def makeFakeImages(xim=None, yim=None, sig1=0.2, sig2=0.2, psf1=2.2, psf2=3.3, offset=None,
-                   psf_yvary_factor=0., varSourceChange=1/50., theta1=0., theta2=0., im2background=0.,
+def makeFakeImages(xim=None, yim=None, var1=0.04, var2=0.04, psf1=3.3, psf2=2.2, offset=None,
+                   psf_yvary_factor=0., varSourceChange=1/50., theta1=0., theta2=0., im1background=0.,
                    n_sources=500, seed=66, verbose=False):
     """! Make two exposures: a template and a science exposure.
     Add random sources of identical flux, with randomly-distributed fluxes and a given PSF, then add noise.
+    In all cases below, index (1) is the science image, and (2) is the template.
     @param xim,yim image pixel coordinates on which to generate the image grid. Default is (-256:256).
-    @param sig1,sig2 std. dev. of noise to be generated on input images. Defalt is 0.2 for both.
+    @param var1,var2 variance of noise to be generated on input images. Default is 0.04 for both.
     @param psf1,psf2 std. dev. of (Gaussian) PSFs for the two images in x,y direction. Default is
-    [2.2, 2.2] and [3.3, 3.3] for im1 and im2 respectively.
+    [3.3, 3.3] and [2.2, 2.2] for im1 and im2 respectively.
     @param offset add a constant (pixel) astrometric offset between the two images
     @param psf_yvary_factor vary the PSF of the science image by this much across the image
```

Comment:
Please note that this variation is only in the y-direction (which will make the name be clear).